docs/integrations/unity-catalog-kuzu.md

```diff
@@ -0,0 +1,217 @@
+| TIMESTAMP                    | TIMESTAMP                                 |
+| TIMESTAMP_NTZ                   | UNSUPPORTED                                 |
+| STRING                   | STRING                                 |
+| BINARY                       | UNSUPPORTED                      |
+| DECIMAL   | DECIMAL                                 |
+
+#### 3. Scan data from table
+
+Finally, we can utilize the `LOAD FROM` statement to scan the `numbers` table. Note that you need to prefix the 
+external `numbers` table with the database alias (in our example `unity`). See the `USE` statement which allows you to
+skip this prefixing for a specific default database.
+
+```sql
+LOAD FROM unity.numbers
+RETURN *
+```
+
+Result:
+
+```
+┌────────┬────────────┐
+│ as_int │ as_double  │
+│ INT32  │ DOUBLE     │
+├────────┼────────────┤
+│ 564    │ 188.755356 │
+│ 755    │ 883.610563 │
+│ 644    │ 203.439559 │
+│ 75     │ 277.880219 │
+│ 42     │ 403.857969 │
+│ 680    │ 797.691220 │
+│ 821    │ 767.799854 │
+│ 484    │ 344.003740 │
+│ 477    │ 380.678561 │
+│ 131    │ 35.443732  │
+│ 294    │ 209.322436 │
+│ 150    │ 329.197303 │
+│ 539    │ 425.661029 │
+│ 247    │ 477.742227 │
+│ 958    │ 509.371273 │
+└────────┴────────────┘
+```
+
+Currently, Kuzu only supports scanning from Delta Lake tables registered in the Unity Catalog.
+
+
+#### 4. `USE` statement
+
+You can use the `USE` statement for attached Unity Catalog to use a default Unity Catalog name (without an alias)
+for future operations.
+This can be used when reading from an attached Unity Catalog to avoid specifying the full catalog name
+as a prefix to the table name.
+
+Consider the same attached Unity Catalog as above:
+
+```sql
+ATTACH 'unity' AS unity (dbtype UC_CATALOG);
+```
+
+Instead of defining the catalog name via `unity.numbers` for each subsequent clause, you can do:
+
+```sql
+USE unity;
+LOAD FROM numbers
+RETURN *
+```
+
+#### 5. Copy data from table
+
+The main purpose of the Unity Catalog extension is to facilitate seamless data transfer from tables in Unity Catalog to Kuzu.
+In this example, we continue using the `unity` database, but this time,
+we copy the data and persist it to Kuzu database. This is done with the `COPY FROM` statement. Here is an example:
+
+We first create a `numbers` table in Kuzu. In this example we will make `numbers` have the same schema as the one defined in the Unity Catalog.
+
+```sql
+CREATE NODE TABLE numbers (id INT32, score DOUBLE , PRIMARY KEY(id));
+```
+
+When the schemas are the same, we can copy the data from the external Unity Catalog table to the Kuzu table simply as follows.
+
+```sql
+COPY numbers FROM unity.numbers;
+```
+If the schemas are not the same, e.g., `numbers` contains only `score` property while the external `unity.numbers` contains
+`id` and `score`, we can still use `COPY FROM` but with a subquery as follows:
+```sql
+COPY numbers FROM (LOAD FROM unity.numbers RETURN score);
+```
+
```

Comment:
We also support copying into a partial subset, right? https://docs.kuzudb.com/import/#copy-from-a-partial-subset. I think we should document that as well. I would do this example:

```
Similarly, you can also specify copying to a subset of the columns in the Kuzu table. For example, if the numbers table in Kuzu's schema was `numbers(id INT32, description String)`, we could copy the `id`'s from the Unity Catalog `numbers` table as follows:
```
```sql
COPY numbers(id) FROM (LOAD FROM unity.numbers RETURN id);
```

Can you do the edit and double check that the command I wrote works?

Reply:
@acquamarin could you please address this?